src/sentry/mediators/external_requests/select_requester.py

```diff
@@ -54,14 +58,20 @@ def _make_request(self):
             )
 
             response = json.loads(body)
-        except Exception:
+        except Exception as e:
+            if self.project:
+                project = self.project.slug
+            else:
+                project = None
+
             logger.info(
                 'select-requester.error',
                 extra={
                     'sentry_app': self.sentry_app.slug,
                     'install': self.install.uuid,
-                    'project': self.project.slug,
+                    'project': project,
```

Comment:
I think you can get rid of the conditional above and just change this to:

```python
'project': self.project and self.project.slug,
```

since it'll already be `None` if not passed.